src/test/regress/expected/citus_non_blocking_splits_no_deferred_drop.out

```diff
@@ -0,0 +1,576 @@
+/*
+This suite runs wihtout deferred drop enabled.
+
+Citus Shard Split Test.The test is model similar to 'shard_move_constraints'.
+Here is a high level overview of test plan:
+ 1. Create a table 'sensors' (ShardCount = 2) to be split. Add indexes and statistics on this table.
+ 2. Create two other tables: 'reference_table' and 'colocated_dist_table', co-located with sensors.
+ 3. Create Foreign key constraints between the two co-located distributed tables.
+ 4. Load data into the three tables.
+ 5. Move one of the shards for 'sensors' to test ShardMove -> Split.
+ 6. Trigger Split on both shards of 'sensors'. This will also split co-located tables.
+ 7. Move one of the split shard to test Split -> ShardMove.
+ 8. Split an already split shard second time on a different schema.
+ 9. Create a colocated table with no replica identity.
+ 10. Show we do not allow Split with the shard transfer mode 'auto' if any colocated table has no replica identity.
+ 11. Drop the colocated table with no replica identity.
+ 12. Show we allow Split with the shard transfer mode 'auto' if all colocated tables has replica identity.
+*/
+CREATE SCHEMA "citus_split_test_schema";
+ALTER SYSTEM SET citus.defer_drop_after_shard_split TO false;
```

Comment:
as far as I can tell, this GUC is only used in the split function itself, so we shouldn't have to use ALTER SYSTEM

There is the citus.defer_shard_delete_interval GUC, which controls how often the deferred drop (and now cleanup) runs. 
That does require ALTER SYSTEM because it is used by the maintenance daemon

Reply:
Got it. Thanks for the explanation. Updated.